ajax/libs/mint-ui/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "mint-ui",
+  "version": "2.2.7",
+  "filename": "mint-ui.common.min.js",
+  "description": "Mobile UI elements for vue.js",
+  "keywords": [
+    "component",
+    "vue",
+    "mobile",
+    "eleme"
```

Comment:
Maybe we can add more keywords, such as `ui-kit`, which under the GitHub repo's description blue tag part.
What do you think?